### PR TITLE
Add the features branch into the main project

### PR DESCRIPTION
**Pull request by flov3rh**
> Linters setup and not showing errors.
> Followed the requirements.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,162 @@
...
+```
+
+### Usage
+
+To run the project, execute the following command simply open the live version or the index.html file.
+
+### Run tests
+
+To run tests, run the following command:
+
```
Comment (Strangeal) [inline README.md]: - [ ] Kindly consider including the steps needed for users to run the test as you have included the text saying they should follow these steps but haven't included the steps.

File: README.md [hunk 2]
```diff
@@ -1 +1,162 @@
...
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
```
Comment (Strangeal) [inline README.md]: - [ ] Kindly consider removing this section of the `README` as it's only needed when your project is going to be deployed. If your project is not going to be deployed it is important that you remove it to keep your `README` file professional, clean, and direct for your users. 
**Also please make sure to remove it from your table of content 👆👆**

File: README.md [hunk 3]
```diff
@@ -1 +1,162 @@
...
+- **[Improve commit messages]** By using the workflow and keeping in mind persons that will need to review the code we write better commit messages.
+- **[Pull Request]** Finally learning how to use branches and pull requests.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://htmlpreview.github.io/github.com/flov3rh/hello-microverse/tree/main)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Strangeal) [inline README.md]: - [ ] Kindly consider removing this section of the `README` as it's only needed when your project is deployed and you have the live link to share with your users. If your project is not deployed it is important that you remove it to keep your `README` file professional, clean, and direct for your users. 
**Also please make sure to remove it from your table of content 👆👆**

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @flov3rh, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights
Well-structured files ✅
Correct use of GitHub Flow ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1 +1,133 @@
...
+
+<!--
+Example command:
+
+```sh
+   npx hint .
+   npx stylelint "**/*.{css,scss}"
+
+```
+--->
```
Comment (skyv26) [inline README.md]: - [x] @flov3rh well done you did make changes as per asked by your previous code reviewer, but there is a small issue and that is your steps that need to follow for running your test are commented out. Kindly remove the comments in order to make your steps visible publicly.

```suggestion
Example command:

```sh
   npx hint .
   npx stylelint "**/*.{css,scss}"

```

**Please remove  from your README for the test section.**

**Review by skyv26 — CHANGES_REQUESTED** [review]
## Hi @flov3rh, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## STATUS: REQUIRE CHANGES 🔴 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 
- Professional `README` template ✅ 
- Linters are green ✅ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by algerina — APPROVED** [review]
Hi @flov3rh ,

Great job so far!  

May I highlight the following: 
- [x] Your  project is setup correctly👏  ✔️ 
- [x] You implemented the requirements of the previous TSE✔️ 





## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:

Your project is complete!  It's time to merge it :shipit:



 
 ## Optional suggestions

 - Please check the comment under the review.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread and tag me @algerina if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

**Pull request merged**